src/OpenTelemetry.AutoInstrumentation/Configurations/LogSettings.cs

```diff
@@ -87,6 +90,16 @@ private static IReadOnlyList<LogExporter> ParseLogExporter(Configuration configu
             }
         }
 
+        if (ConsoleExporterEnabled)
```

Comment:
Can we do it outside this method so that it remains `static`?
Same comment applies to other places.

Reply:
I would say that this code is cool enough, not worth to make it better. It should be shortly with us. Probably only 2-3 more releases.